src/components/ModalExport.native.tsx

```diff
@@ -360,4 +402,21 @@ const styles = StyleSheet.create({
   },
 })
 
-export default ModalExport
```

Comment:
https://github.com/cybersemics/em/blob/05142c9b35f60f67b10366f9fe2bd1b6f1fbc185/src/components/ModalExport.tsx#L137-L150

You have missed adding this component. `ExportThoughtsPhrase` updates the number of descendants in real time when they are being pulled. 

https://github.com/cybersemics/em/blob/05142c9b35f60f67b10366f9fe2bd1b6f1fbc185/src/components/ModalExport.tsx#L418